### PR TITLE
refactor(client/ios): move fetching logic from TypeScript to Go

### PR DESCRIPTION
**Pull request by jyyi1**
#### New Fetch Error

image

Prerequisite: #2220, #2221

### CODE REVIEW

File: client/src/cordova/plugin/apple/src/OutlinePlugin.swift [hunk 1]
```diff
@@ -281,6 +301,11 @@ class OutlinePlugin: CDVPlugin {
       send(pluginResult: result, callbackId: callbackId, keepCallback: keepCallback)
   }
 
+  private func sendSuccess(_ operationResult: String, callbackId: String, keepCallback: Bool = false) {
+    let result = CDVPluginResult(status: CDVCommandStatus_OK, messageAs: operationResult)
+    send(pluginResult: result, callbackId: callbackId, keepCallback: keepCallback)
```
Comment (fortuna) [inline client/src/cordova/plugin/apple/src/OutlinePlugin.swift:306]: For clarity:
```suggestion
    self.send(pluginResult: result, callbackId: callbackId, keepCallback: keepCallback)
```

File: client/src/www/app/main.cordova.ts [hunk 2]
```diff
@@ -118,11 +118,7 @@ class CordovaPlatform implements OutlinePlatform {
   }
 
   getResourceFetcher(): ResourceFetcher {
-    if (cordova.platformId === 'android') {
-      return new CordovaResourceFetcher();
-    }
-    // TODO: move to Go fetch implementation later
-    return new BrowserResourceFetcher();
+    return new CordovaResourceFetcher();
```
Comment (fortuna) [inline client/src/www/app/main.cordova.ts:121]: Need to support Android first.

**Review by fortuna — APPROVED** [review]

File: client/src/www/app/main.cordova.ts (hunk 2 above)
Comment (jyyi1) [reply client/src/www/app/main.cordova.ts:121]: Android support has been merged

**Comment by jyyi1** [comment]
@fortuna I rebased this branch to master, take another look?

**Pull request merged**